src/panels/lovelace/editor/config-elements/hui-weather-forecast-card-editor.ts

```diff
@@ -88,6 +94,39 @@ export class HuiWeatherForecastCardEditor
         },
         ...(hasForecast
           ? ([
+              {
+                name: "forecast_type",
+                selector: {
+                  select: {
+                    options: [
+                      {
+                        value: "legacy",
+                        label: localize(
+                          "ui.panel.lovelace.editor.card.weather-forecast.no_type"
+                        ),
+                      },
+                      {
+                        value: "daily",
+                        label: localize(
+                          "ui.panel.lovelace.editor.card.weather-forecast.daily"
+                        ),
+                      },
+                      {
+                        value: "hourly",
+                        label: localize(
+                          "ui.panel.lovelace.editor.card.weather-forecast.hourly"
+                        ),
+                      },
+                      {
+                        value: "twice_daily",
+                        label: localize(
+                          "ui.panel.lovelace.editor.card.weather-forecast.twice_daily"
```

Comment:
We should only show these options if the chosen entity supplies them

Reply:
Any help here. I can't figure out how to do it now for a few good hours.

Reply:
Check how it is done for the forecast options: <https://github.com/home-assistant/frontend/blob/dev/src/panels/lovelace/editor/config-elements/hui-weather-forecast-card-editor.ts#L89>

Reply:
```js
...(supportsHourly ? [{
                        value: "hourly",
                        label: localize(
                          "ui.panel.lovelace.editor.card.weather-forecast.hourly"
                        ),
                      }]
    : [])
```